common/lib/dependabot/clients/azure.rb

```diff
@@ -183,6 +183,28 @@ def create_pull_request(pr_name, source_branch, target_branch,
           "/_apis/git/repositories/" + source.unscoped_repo +
           "/pullrequests?api-version=5.0", content.to_json)
       end
+
+      def pull_request(pull_request_id)
+        response = get(source.api_endpoint +
+          source.organization + "/" + source.project +
+          "/_apis/git/pullrequests/" + pull_request_id)
+
+        JSON.parse(response.body)
```

Comment:
Not blocking for now, but we might want to handle non-200 responses here

Reply:
I see we have some sort of handling in the get method of the azure client. Can you specify the scenarios that we might want to cover here?